src/components/screens/multisigAccountDetails/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from 'react';
+import { withTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+
+import Box from '../../toolbox/box';
+import BoxHeader from '../../toolbox/box/header';
+import BoxContent from '../../toolbox/box/content';
+import BoxInfoText from '../../toolbox/box/infoText';
+import Dialog from '../../toolbox/dialog/dialog';
+import Tooltip from '../../toolbox/tooltip/tooltip';
+import { Members } from '../multiSignature/summary/summary';
+import { getActiveTokenAccount, extractAddress } from '../../../utils/account';
+import styles from './styles.css';
+
+const MultisigAccountDetails = ({ t }) => {
+  const account = useSelector(state => getActiveTokenAccount(state));
```

Comment:
I have started to follow the general structure of having an index file for every exported component and then using higher order components or hooks that I don't wanna be concerned with called in that index file so that the unit tests for that particular component don't have any details of how the hook/hocs are implemented. This makes the unit tests clearer and easier to write/debug. 

In general, we should start using the ES module system more i.e. within folders, have an index file that exposes modules out of the that folder and code inside that folder should only be accessed through that index file. This also helps with tree shaking and lazy loading.

Reply:
Thought it's a good point for making the tests easier, but it also scatters the code in more files. I wasn't aware of this decision and I requested contradicting changes on #3130 from @isalga .